packages/utils/src/viewHelpers.ts

```diff
@@ -0,0 +1,22 @@
+// A collection of helper methods for user-facing views
+
+// Accepts `seconds` as a single argument and returns a
+// human-readable string of a duration of time.
+// ie: 1 day, 3 hours, 4 minutes, 20 seconds
+export function getReadableDuration(seconds: number): string {
+  const out = [];
+  let period: [number, string];
+  const periods: Array<[number, string]> = [[86400, "day"], [3600, "hour"], [60, "minute"], [1, "second"]];
+  let secondsRemaining: number = seconds;
+
+  while (periods.length) {
```

Comment:
you could simplify this with reduce.

```
let secs: number = this.state.secondsRemaining;
return periods.reduce((acc: string, item: [number, string], index: number) => {
   const periodUnits = Math.floor(secs / item[0]);
   if (periodUnits > 0) {
      acc += `${periodUnits} ${item[1]}${periodUnits !== 1 ? "s" : ""}${index < periods.length -1 ? " ": ""}`;
      secs = secs % item[0];
   }
   return acc;
}, "");
```

Reply:
I forgot about template literals being available. This is a good suggestion.. I'm going to refactor my code.

Reply:
you can use almost everything